common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -875,6 +875,23 @@ def user_details_force_sync(auth_entry, strategy, details, user=None, *args, **k
                 changed[provider_field] = current_value
                 setattr(model, field, provider_value)
 
+        # Generate fullname only for IES IDP.
+        try:
+            if (not details.get('fullname') and details.get('first_name') and details.get('last_name') and
+                any(['iam-stage.pearson.com' in current_provider.entity_id, 'iam.pearson.com' in current_provider.entity_id])):  # pylint: disable=line-too-long
+                fullname_value = '%s %s' % (details.get('first_name'), details.get('last_name'))
```

Comment:
Please, use format to concatenate the strings. @ivanvgh 
There is not necessary to create this variable since as it's only used once.

Reply:
Concatenated using format, it is necessary to create fullname_value variable since it is used twice.